File: QualityTest/src/main/java/MainListner.java
import Messages.FeedbackProducer;
import Messages.MessageListner;
import Messages.MetricsHandler;
import Messages.RequestHandler;
import java.io.IOException;
import java.util.concurrent.TimeoutException;

public class MainListner {
  public static void main(String[] args) throws IOException, TimeoutException {
    FeedbackProducer feedbackProducer = new FeedbackProducer("feedback","rabbitmq");
    RequestHandler handler = new MetricsHandler(feedbackProducer);
    MessageListner listner = new MessageListner("metrics","rabbitmq",handler);

    listner.listen();
  }
}

File: ClassIdentifier/mainListner.py
import os
import sys

from Messages.FeedbackProducer import FeedbackProducer
from Messages.RequestHandler import RequestHandler
from Messages.RequestListner import RequestListner

Feedback = FeedbackProducer("rabbitmq","feedback")
Handler = RequestHandler(Feedback)
Listner = RequestListner("rabbitmq","history",Handler)


if __name__ == '__main__':
    try:
        print("Listening")
        Listner.listen()
    except  KeyboardInterrupt as e :
        print(e)
        try:
            sys.exit(0)
        except SystemExit:
            os._exit(0)
